packages/ecmascript/src/Set.ts

```diff
@@ -13,7 +13,7 @@
  * #L%
  */
 export default class Set<T extends string | number> {
-	private size = 0;
+	private size: number = 0;
```

Comment:
Shouldn't size be a public property?

Reply:
Actually not, as in the resulting code it would be public anyway.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/size Same comment